src/test/resources/templates/ERT000002.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version = '1.0' encoding = 'UTF-8'?><template xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jweb-1a.ebi.ac.uk:21040/embl/genomes/submission/template.xsd" id="ERT000002" version="3">
```

Comment:
- Comment: can we change the code to pull the resources from the main and remove the copies from the test. This would improve the unit test.